src/debug/di/rsfunction.cpp

```diff
@@ -570,6 +570,38 @@ HRESULT CordbFunction::GetActiveReJitRequestILCode(ICorDebugILCode **ppReJitedIL
     return hr;
 }
 
+//-----------------------------------------------------------------------------
+// CordbFunction::CreateNativeBreakpoint
+//  Public method for ICorDebugFunction4::CreateNativeBreakpoint.
+//   Sets a breakpoint at offset 0 for all native code versions of a method.
```

Comment:
To be really clear I would say 'Sets a breakpoint at native offset 0'.

Reply:
Thanks